rclcpp/include/rclcpp/parameter_client.hpp

```diff
@@ -268,6 +268,66 @@ class SyncParametersClient
 };
 
 }  // namespace parameter_client
+
+
+class ParameterEventsFilter
+{
+public:
+  RCLCPP_SMART_PTR_DEFINITIONS(ParameterEventsFilter)
+  enum class EventType {NEW, DELETED, CHANGED};  ///< An enum for the type of event.
+  /// Used for the listed results
+  using EventPair = std::pair<EventType, rcl_interfaces::msg::Parameter *>;
+
+  RCLCPP_PUBLIC
+  inline ParameterEventsFilter(
```

Comment:
I believe the use of `inline` here (and below) is redundant:

> A function defined entirely inside a class/struct/union definition, whether it's a member function or a non-member friend function, is implicitly an inline function.

-- http://en.cppreference.com/w/cpp/language/inline